account-kit/react/src/components/navigation.tsx

```diff
@@ -27,7 +27,11 @@ export const Navigation = ({
             : "invisible"
         }
       >
-        <BackArrow />
+        <BackArrow
+          width="22"
+          height="22"
+          className="w-[24px] h-[24px] lg:w-[16px] lg:h-[16px]"
```

Comment:
I get the `className` dimensions, but why do we also need the height and width on the SVG

Reply:
Great catch! Resolved.